src/graphql/schema/Playlist/playlist.js

```diff
@@ -1,18 +1,56 @@
 import { gql } from 'apollo-server-express';
+import Playlist from '../../../models/Playlist';
 
 // Type definitions for Playlists go here
 export const typeDefs = gql`
   extend type Query {
-    playlists: [Playlist!]!
+    getPlaylists: [Playlist]!
   }
+
+  extend type Mutation {
+    createPlaylist(name: String!, description: String): Playlist
+    addTracks(playlistId: String, tracks: [String]): SnapshotID
```

Comment:
Same question as above.  Just curious what this does

Reply:
This is the mutation portion of the resolvers. This is where you put methods that will alter the data in some way. The first method I explained in the previous comment. The addTracks method should take in a playlistId and an array of tracks (I believe they'll be strings that are ids) and the type returned will be a SnapshotID that happens to be a string returned from the API. It is still a work in progress and hope it works the way I expect it to, lol.